src/features/user/userSlice.js

```diff
@@ -24,8 +24,15 @@ const userSlice = createSlice({
     updateDP: (state, action) => {
       state.user = { ...state.user, photoURL: action.payload };
     },
+    setFollowers: (state, action) => {
```

Comment:
Would you need these? Since you'll be deriving them from the user anyways.

Reply:
Thanks for pointing this out. I missed removing it after I made the changes.